src/test/java/org/gridsuite/study/server/LoadflowTest.java

```diff
@@ -427,7 +427,7 @@ public void testOverloadedLines() throws Exception {
         UUID modificationNode1Uuid = modificationNode1.getId();
 
```

Comment:
you can also rename the test function: testCurrentLimitViolations()

Reply:
Indeed.Done